### PR TITLE
added 2 new API args

### PR DESCRIPTION
**Pull request by chrisbc**

### CODE REVIEW

File: src/main/java/nz/cri/gns/NZSHM22/opensha/inversion/NZSHM22_AbstractInversionRunner.java [hunk 1]
```diff
@@ -342,7 +383,10 @@ public FaultSystemSolution runInversion() throws IOException, DocumentException
 		tsa.setConstraintRanges(inversionInputGenerator.getConstraintRowRanges());
 		tsa.setRandom(new Random(1));
 		tsa.setRuptureSampler(null);
-		tsa.setPerturbationFunc(GenerationFunctionType.UNIFORM_NO_TEMP_DEPENDENCE);
+//		tsa.setPerturbationFunc(GenerationFunctionType.UNIFORM_NO_TEMP_DEPENDENCE);
+
+		tsa.setPerturbationFunc(perturbationFunction);
+//		tsa.setNonnegativeityConstraintAlgorithm(nonNegAlgorithm);
```
Comment (voj) [inline src/main/java/nz/cri/gns/NZSHM22/opensha/inversion/NZSHM22_AbstractInversionRunner.java:389]: are you sure you don't want to use the new property here?

**Review by voj — COMMENTED** [review]
Looks good, not sure if line 389 is commented out on purpose

File: src/main/java/nz/cri/gns/NZSHM22/opensha/inversion/NZSHM22_AbstractInversionRunner.java (hunk 1 above)
Comment (chrisbc) [reply src/main/java/nz/cri/gns/NZSHM22/opensha/inversion/NZSHM22_AbstractInversionRunner.java:389]: bugger - well spotted thanks!

Comment (voj) [reply src/main/java/nz/cri/gns/NZSHM22/opensha/inversion/NZSHM22_AbstractInversionRunner.java:389]: yay for code reviews!

**Pull request merged**